### PR TITLE
Header and modals

### PR DESCRIPTION
**Pull request by mkandler**
- Builds out scrolled and mobile states for header
- Desktop version of the team/provider modals
- Styles for privacy and terms pages

### CODE REVIEW

File: apps/marketing/components/Header/index.js [hunk 1]
```diff
@@ -7,31 +8,98 @@ const cx = classNames.bind(styles)
...
+    this.toggleOpen = this.toggleOpen.bind(this)
+    this.handleScroll = this.handleScroll.bind(this)
+    this.state = {
+      open: false
+    }
+    this.scrollActivated = false
+  }
+
+  componentDidMount () {
+    if (typeof window === 'undefined') {
```
Comment (craigspaeth) [inline apps/marketing/components/Header/index.js]: 👍 nice universal JS guard

File: apps/marketing/components/Header/index.js [hunk 2]
```diff
@@ -7,31 +8,98 @@ const cx = classNames.bind(styles)
...
+      return
+    }
+    window.addEventListener('scroll', this.handleScroll)
+    setTimeout(() => {
+      this.handleScroll()
+    }, 50)
+    // Close header if open on modal and be sure to scroll to top
```
Comment (craigspaeth) [inline apps/marketing/components/Header/index.js]: 🙇 appreciate the comment here

File: apps/marketing/pages/coaching.js [hunk 3]
```diff
@@ -62,9 +69,12 @@ class Coaching extends Component {
       overview,
       issues,
       comparison,
+      why,
       numberedSection
     } = this.props
 
+    console.log(overview)
```
Comment (craigspaeth) [inline apps/marketing/pages/coaching.js]: Stray `console.log`

**Comment by craigspaeth** [comment]
Solid! LGTM minus the stray `console.log`. Feel free to merge when you remove that. Thanks!

File: apps/marketing/pages/coaching.js (hunk 3 above)
Comment (mkandler) [reply apps/marketing/pages/coaching.js]: I need to write a local script to catch these... this happens way too often to me 😅

File: apps/marketing/components/Header/index.js (hunk 2 above)
Comment (mkandler) [reply apps/marketing/components/Header/index.js]: Good to know! I like leaving comments on unintuitive lines of code, but I work with some teams that are 100% anti-commenting

**Pull request merged**